2018/iaam5/README.md

```diff
@@ -70,7 +69,7 @@ We have successfully deployed our old application in Kubernetes and scaled it bu
 
 # Deploy an external database (./iaam5-db)
 
-- ./iaam5-db/db.yml holds the YAML definition for our Kubernetes deployment of the database. As you can see we make the server available with the port 5432. We are also using a ConfigMap to configure the initialize the database and we can use this config map to later configure the application. There is alos a readiness probe in our container definition which tells Kubernetes that the container is ready to be accessed. Since we do not want to access the database from outside of our cluster we do not expose the service to the outside but keep an internal service which is also defined in the same file.
+- ./iaam5-db/db.yml holds the YAML definition for our Kubernetes deployment of the database. As you can see we make the server available with the port 5432. We are also using a ConfigMap to configure the database and we can use this ConfigMap to later configure the application. There is alos a readiness probe in our container definition which tells Kubernetes that the container is ready to be accessed. Since we do not want to access the database from outside of our cluster we do not expose the service to the outside but keep an internal service which is also defined in the same file.
```

Comment:
please fix the typo: alos -> also
additionally please put a colon before the "but"